packages/client-sdk-nodejs/package-lock.json

```diff
@@ -42,6 +42,7 @@
       }
     },
     "../common-integration-tests": {
+      "name": "@gomomento/common-integration-tests",
```

Comment:
added the lines in this file back in and the integration tests pass now, but should these be here?

Reply:
int tests failed again actually, and these don't seem to be necessary so going to remove them

Reply:
you should never edit this file by hand. if it ever gets messed up, just delete it, and run `npm install`.